samples/go/downloader/downloader.go

```diff
@@ -0,0 +1,322 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
```

Comment:
i'll leave it here until we're done with the Code Review and then move it to attic at that point.